docs/source/platform/federation.mdx

```diff
@@ -0,0 +1,228 @@
+```
+
+Navigate to the Services tab in the Apollo Platform to see an interactive list of information about the implementing services in your graph.
+
+<div style="text-align:center">
+    <img src="../images/ServicesListPlatform.png" alt="Service List">
+    </img>
+</div>
+
+[//]: # (Description: How to see the current services in your graph. This section might be expanded later to discuss showing history, composition results, etc. This section should include some screenshots)
+[//]: # (Assignee: Maya)
+[//]: # (Reviewer: Justin)
+
+## Validating changes to your graph
+
+[//]: # (explain how schema validation works with federation, highlighting that there's an extra step in the `apollo service:check` command which will build the composed schema to run traffic validation against. Ideally, this change could be done as a toggle either for each section or for the entire page where the extra details and steps are added for federation. If the toggle is difficult, then we should simply have a pattern that we use, like using a different color or an italicized font for the parts of the process that are federation-specific)
+[//]: # (Assignee: Caydie)
+[//]: # (Reviewer: Jake)
+
+Federation allows teams to work independently on implementing services without needing to coordinate over a gigantic and all-encompassing schema. However, this increase in autonomy requires control to ensure that teams that operate on different services are respecting [defined dependencies](https://www.apollographql.com/docs/apollo-server/federation/federation-spec/). The Apollo GraphQL platform provides tools to help ensure that this increase in autonomy doesn't come at a cost to stability.
+
+In particular, along with [validating overall schema changes against known operations](https://www.apollographql.com/docs/platform/schema-validation/), running `apollo service:check` for an implementing service will ensure that the overall Graph still composes to a valid schema, and will output any violated dependencies if present.
+
+With a federated graph, use the `apollo service:check` command to validate individual service changes by adding the `--serviceName` flag. Engine will run composition on the proposed partial schema with the current list of implementing services and their partial schemas, making sure that the composition is successful. That composed schema will then be diff'ed against the most recently registered schema and validate that those changes are safe.
+
+
+> Using the `apollo service:push` and `apollo service:check` command to track changes to your graph
+
+<!--
+[//]: # (Description: An overview of how and when to use `apollo service:push` and `apollo service:check` in your CI/CD workflow, linking out to the specifics of usage in the specific docs (#502 and #503). Ideally this section includes some screenshots of the workflows in practice.)
+[//]: # (Assignee: James)
+[//]: # (Reviewer: Adam Z)
+-->
+
+## Running with managed federation
```

Comment:
Rename to Managed Federation

This section should bring in all of the details from the below managed federation section and likely have some sub-sections, with an introduction.